contracts/strategies/strategy-joe-rush-farm-base.sol

```diff
@@ -126,4 +126,34 @@ abstract contract StrategyJoeRushFarmBase is StrategyBase {
             _snob.sub(_share)
         );
     }
+
+    function _swapBaseToToken(uint256 _amount, address token1, address token2) internal {
+        address[] memory path = new address[](3);
+        path[0] = token1;
+        path[1] = wavax;
+        path[2] = token2;
+        IERC20(token1).safeApprove(joeRouter, 0);
+        IERC20(token1).safeApprove(joeRouter, _amount);
+        _swapTraderJoeWithPath(path, _amount);
+    }
+
+    function _takeFeeRewardToSnob(uint256 _keep, address reward) internal {
```

Comment:
Could we remove `_takeFeeJoeToSnob()` (line 113) and use this function instead for Joe rewards? Functionality is identical.

Reply:
no, because the _takeFeeJoeToSnob also has revenueShare, feeDistributor logic, etc.

Reply:
this requires testing every contract that inherits from the this base

Reply:
You could also simplify the body of _takeFeeJoeToSnob(_reward) to just call takeFeeRewardToSnob(joe, _reward).  Then child/other contracts can still call either method.